pkg/validator/cluster.go

```diff
@@ -911,6 +938,13 @@ func admitClusterCR(
 		}
 	}
 
+	// If connections, be it cluster or configmap
+	// is updated then increment configMetaGenerator to trigger
+	// regeneration of configmeta
+	if !reflect.DeepEqual(clusterCR.Spec.Connections, prevClusterCR.Spec.Connections) {
+		clusterCR.Spec.ConfigMetaGenerator = prevClusterCR.Spec.ConfigMetaGenerator + 1
+	}
```

Comment:
I don't think we need to change the in-memory CR here... none of the following validator code is going to look at it. The new validateConnections function you added will take care of patching the resource stored in etcd.

Reply:
If I remember this correctly then, if a new configmap is added or if an existing configmap is deleted to the connections then we would need this

Reply:
The value written to here is not used by anything.

Reply:
Yeah , its not required i guess